aquamarine-vm/src/errors.rs

```diff
@@ -29,6 +30,12 @@ pub enum AquamarineVMError {
 
     /// Errors related to stepper execution.
     StepperError(StepperError),
+
+    /// I/O errors while persisting resulted data.
+    PersistDataError(IOError),
+
+    /// Errors related to particle_data_store path from supplied config.
+    InvalidDataStorePath(IOError),
```

Comment:
I think it's crucial to display the path error happened on. Otherwise it is pretty hard to act on that error, since you don't really know the reason